src/main/scala/com/sourcegraph/sbtsourcegraph/Versions.scala

```diff
@@ -65,6 +66,67 @@ object Versions {
       .updated(semanticdbJavacKey, semanticdbJavacVersions.last)
   }
 
+  private val jvmVersionCache = collection.mutable.Map.empty[Option[File], Int]
+
+  def isJavaAtLeast(n: Int, home: Option[File] = None): Boolean = {
+
+    val significant = jvmVersionCache.getOrElseUpdate(
+      home, {
+        val raw =
+          home match {
+            case None =>
+              System.getProperty("java.version")
+            case Some(javaHome) =>
+              val sb = new StringBuilder
+              val proc = {
+                val cmd =
+                  if (scala.util.Properties.isWin)
+                    Paths.get("bin", "java")
+                  else Paths.get("bin", "java")
+
+                scala.sys.process
+                  .Process(Seq(cmd.toString(), "-version"), cwd = javaHome)
```

Comment:
This is always going to be quite brittle. Ideally, we use something like `SystemJavaVersion` from scip-java. Not a blocking comment, just an FYI if you don't want too maintain this parser forever https://sourcegraph.com/github.com/sourcegraph/scip-java/-/blob/scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/SystemJavaVersion.scala?L16

Reply:
That sounds good.